### PR TITLE
Change tests to reference demo website

### PR DESCRIPTION
**Pull request by emalysz**
See https://browserstack.atlassian.net/browse/DOC-2?focusedCommentId=304255

### CODE REVIEW

**Comment by emalysz** [comment]
@francisf

File: specs/parallel_test.js [hunk 1]
```diff
@@ -20,18 +20,38 @@ parallel('Tests ', function() {
...
+                  // locating product in cart and getting name of the product in cart
+                  driver.findElement(webdriver.By.xpath('//*[@id="__next"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]')).getText().then(async function (productCartText) {
+                    try {
+                      // checking whether product has been added to cart by comparing product name
+                      assert(productText === productCartText);
+                      //marking the test as Passed if product has been added to the cart
+                      await driver.executeScript(
+                        'browserstack_executor: {"action": "setSessionStatus", "arguments": {"status":"passed","reason": "Product has been successfully added to the cart!"}}'
+                      );
+                      driver.quit().then(function() {
```
Comment (francisf) [inline specs/parallel_test.js]: this should be in finally block?  +

done() in single_test.js

File: specs/parallel_test.js [hunk 2]
```diff
@@ -20,18 +20,38 @@ parallel('Tests ', function() {
...
+              driver.findElement(webdriver.By.xpath('//*[@id="1"]/div[4]')).click().then(function() {
+                // waiting until the Cart pane has been displayed on the webpage
+                driver.findElement(webdriver.By.className('float-cart__content')).then(function () {
+                  // locating product in cart and getting name of the product in cart
+                  driver.findElement(webdriver.By.xpath('//*[@id="__next"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]')).getText().then(async function (productCartText) {
+                    try {
+                      // checking whether product has been added to cart by comparing product name
+                      assert(productText === productCartText);
+                      //marking the test as Passed if product has been added to the cart
+                      await driver.executeScript(
```
Comment (francisf) [inline specs/parallel_test.js]: can we use await everywhere (vs promises - then) + in single_test.js

File: conf/parallel.conf.js [hunk 3]
```diff
@@ -3,17 +3,39 @@ var config = {
...
-    },{
-      'browserName': 'Safari'
-    },{
-      'browserName': 'Firefox'
-    },{
-      'browserName': 'IE'
+      'os_version': '10',
+      'browserName': 'Chrome',
+      'browser_version': 'latest',
+      'os': 'Windows',
```
Comment (francisf) [inline conf/parallel.conf.js]: minor nitpick here and for other caps.. keep os and os_version keys next to each other.

**Review by francisf — CHANGES_REQUESTED** [review]

**Comment by emalysz** [comment]
@francisf committed some changes to the branch

**Review by francisf — APPROVED** [review]

**Pull request merged**